appcelerator-archive/amp
# add stack-id and stack-name as labels in containers

related to #232
add stack id and stack name as labels in container to allow amp-agent to add them in logs

test:
- make, start swam amp
- create a stack: amp stack up test -f [stack file]
- verify using docker inspect than containers created for stack services have labels: io.amp.stack.id and io.amp.stack.name

regression tests: make test
